packages/core/src/types.ts

```diff
@@ -284,6 +287,8 @@ export interface StateValueMap {
 /**
  * The string or object representing the state value relative to the parent state node.
  *
+ * @remarks
```

Comment:
I'm still quite unsure why we need this. RxJS has *extensive* and long JSDocs and they use api-extractor too but there is no `@remarks` in sight in their codebase. I wonder what's the difference? 🤔

Reply:
On the index page a description is provided for each item.

With `@remarks`, we can choose to show only the first summary line in the descriptions:
![index-with-remarks](https://github.com/statelyai/xstate/assets/23459/f292cc99-3ecc-4c1d-ba41-e634197091c1)

Without `@remarks`, the description is longer (and most of the index descriptions are a single line so this one stands out):
![index-without-remarks](https://github.com/statelyai/xstate/assets/23459/12227c91-72cb-4000-8db7-9c3cfe286f4e)

So it’s a subjective change, but I thought it was better to have a shorter description. If you disagree I’m happy to change it back.

I had also noticed the list items weren’t formatted properly on the index, so I wanted to omit them, but I just found a fix for that I can push to this PR, so it's not a deciding factor.

Reply:
API Extractor uses TSDoc, and `@remarks` is specific to TSDoc. See: https://tsdoc.org/pages/tags/remarks

Reply:
That's a good point. I’ll see what I can find out about TSDocs in VS Code. Might be some ways to improve the experience.